README.md

```diff
@@ -2,302 +2,708 @@
 [![Build Status](https://travis-ci.org/ismasan/parametric.png)](https://travis-ci.org/ismasan/parametric)
 [![Gem Version](https://badge.fury.io/rb/parametric.png)](http://badge.fury.io/rb/parametric)
 
-DSL for declaring allowed parameters with options, regexp pattern and default values.
+Utilities for declaring and manipulating attribute schemas, including custom validations and coercions.
```

Comment:
This sentence confused me when I first started reading, because I didn't know what attribute schemas were. I suspect it might be better to start by explaining what the gem is for, rather than what it does?